tests/Unit/CTS/ValidationPositionRepositoryTest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Tests\Unit\CTS;
+
+use App\Models\Cts\Member;
+use App\Models\Cts\Validation;
+use App\Models\Cts\ValidationPosition;
+use App\Repositories\Cts\ValidationPositionRepository;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+use Tests\TestCase;
+
+class ValidationPositionRepositoryTest extends TestCase
+{
+    use DatabaseTransactions;
+
+    /* @var ValidationPositionRepository */
+    protected $subjectUnderTest;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->subjectUnderTest = resolve(ValidationPositionRepository::class);
+    }
+
+    /** @test */
+    public function itCanFindAPositionById()
+    {
+        $position = factory(ValidationPosition::class)->create();
+        $search = $this->subjectUnderTest->findByPositionId($position->id);
+
+        $this->assertEquals($position->id, $search->id);
+    }
+
+    /** @test */
+    public function itCanFindAPositionByCallsign()
+    {
+        $position = factory(ValidationPosition::class)->create([
+            'position' => 'Shanwick (EGGX_FSS)'
+        ]);
+
+        $search = $this->subjectUnderTest->findByPosition('EGGX_FSS');
+
+        $this->assertEquals($position->id, $search->id);
+    }
+
+    /** @test */
+    public function itReturnsValidatedMembers()
+    {
+        $position = factory(ValidationPosition::class)->create();
+
+        factory(Validation::class, 10)->create([
+            'position_id' => $position->id
+        ]);
+
+        $this->assertCount(10, $position->members);
+    }
+
+    /** @test */
+    public function itFormatsValidatedMembers()
+    {
+        $position = factory(ValidationPosition::class)->create();
+        $member = factory(Member::class)->create();
+
+        factory(Validation::class)->create([
+            'position_id' => $position->id,
+            'member_id' => $member->id
+        ]);
+
+        $expected = [
```

Comment:
Is this not just `$member->toArray()`?

Reply:
Nope because that would pull back all of the other keys associated with a member. This test is very explicit in ensuring we only expose the member's name and id.